gating/pre_merge_test/post_send_junit_to_qtest.sh

```diff
@@ -10,6 +10,10 @@ export QTEST_API_TOKEN=$RPC_ASC_QTEST_API_TOKEN
 VENV_NAME="venv-qtest"
 PROJECT_ID="76551"
 
+# Work-around for ASC-592. Hardcoded for proper results in qtest
+export RPC_PRODUCT_RELEASE="newton"
+export RPC_RELEASE="r13.?.?"
```

Comment:
@johnduarte @prometheanfire Where can I find the correct version number to set this to for newton and newton-rc?

Reply:
I thought newton was r14, not sure where that's doc'd.

Reply:
As pointed out by @mattt416 in slack, the release version is tracked within the repository by the `release.yml` file https://github.com/rcbops/rpc-openstack/blob/newton/group_vars/all/release.yml and sourced by the `functions.sh` script https://github.com/rcbops/rpc-openstack/blob/newton/scripts/functions.sh#L59-L60

There appears to be a bug. The `functions.sh` script _is_ sourced from within the `run_deploy.sh` script https://github.com/rcbops/rpc-openstack/blob/newton/gating/pre_merge_test/run_deploy.sh#L30-L32, but is _not_ sourced from within the `run_deploy_mnaio.sh` script https://github.com/rcbops/rpc-openstack/blob/newton/gating/pre_merge_test/run_deploy_mnaio.sh

So, the fix for setting the value for `RPC_RELEASE` would be to source the `functions.sh` file from within `run_deploy_mnaio.sh`.